sdk/core/azure-json/src/main/java/com/azure/json/package-info.java

```diff
@@ -2,6 +2,533 @@
 // Licensed under the MIT License.
 
 /**
- * Contains general classes for handling JSON.
+ * <p>The Azure JSON library provides interfaces for stream-style JSON reading and writing. Stream-style reading and
+ * writing has the type itself define how to read JSON to create an instance of itself and how it writes out to JSON.
+ * Azure JSON also allows for external implementations for JSON reading and writing by offering a service provider
+ * interface to load implementations from the classpath. However, if one is not found, the Azure JSON library provides
```

Comment:
Consider adding a link to service provider interface. https://docs.oracle.com/javase/tutorial/sound/SPI-intro.html